### PR TITLE
Improve kubelet logging around startup.

### PR DESCRIPTION
**Pull request by vishh**
This can help debug #6548

### CODE REVIEW

File: pkg/kubelet/server.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ type TLSOptions struct {
 
 // ListenAndServeKubeletServer initializes a server to respond to HTTP network requests on the Kubelet.
 func ListenAndServeKubeletServer(host HostInterface, address net.IP, port uint, tlsOptions *TLSOptions, enableDebuggingHandlers bool) {
-	glog.V(1).Infof("Starting to listen on %s:%d", address, port)
+	glog.Infof("Starting to listen on %s:%d", address, port)
```
Comment (dchen1107) [inline pkg/kubelet/server.go]: Shouldn't this by default on for kubelet since our default verbose level is 2?

**Comment by dchen1107** [comment]
LGTM with small nit

File: pkg/kubelet/server.go (hunk 1 above)
Comment (vishh) [inline pkg/kubelet/server.go]: I don't see vlog being set everywhere. For example hack/test-cmd.sh does not set vlog.

**Comment by vishh** [comment]
@dchen1107 : PTAL

**Comment by dchen1107** [comment]
LGTM

**Pull request merged**